docs/reference/announcements-release-notes/880/880-release-notes.md

```diff
@@ -19,6 +19,129 @@ These release notes identify the new features included in 8.8, including [alpha
+
+<!-- https://github.com/camunda/product-hub/issues/2710 -->
+
+### One Camunda 8 API <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span><span class="badge badge--medium" title="This feature affects APIs">API</span>
+
+Introduces a unified Camunda 8 REST API that consolidates multiple fragmented APIs into a single, coherent interface, simplifying development and improving clarity across Camunda components.
+
+<!-- https://github.com/camunda/product-hub/issues/2243 -->
+
+### Provide resilient batch operations managed by Zeebe <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span><span class="badge badge--medium" title="This feature affects Zeebe">Zeebe</span>
+
+All batch operations, such as cancelling or resolving incidents in bulk, are now handled by Zeebe instead of Operate. This change ensures region failovers in the multi-region setup no longer risk losing critical batch commands. Users will initiate and manage batch operations through the Orchestration Cluster REST API and the Operate UI, but the underlying processing occurs within Zeebe. By moving batch operations to the core engine, multi-region deployments gain reliability and resilience.
+
+<!-- https://github.com/camunda/product-hub/issues/2420 -->
+
+### Run process segment <span class="badge badge--long" title="This feature affects Developer">Developer</span><span class="badge badge--medium" title="This feature affects connectors">Connector</span>
+
+Enables developers to manually execute and test individual tasks or segments (connectors, RPA bots, IDP extractions) without running full processes, improving debugging and development efficiency.
+
+<!-- https://github.com/camunda/product-hub/issues/2453 -->
+
+### Unlock element template fields <span class="badge badge--long" title="This feature affects Developer">Developer</span>
+
+[Element template management](/components/connectors/manage-connector-templates.md) is now more flexible for developers and DevOps teams. You can assign custom semantic IDs and use an intuitive versioning scheme, ensuring templates are portable and retain stable references across different environments. Template names and file names can be managed independently, and you can quickly import templates using copy and paste, git sync, or CI/CD pipeline. Safeguards now notify you of ID or version conflicts to prevent accidental overwrites when publishing templates.
+
+<!-- https://github.com/camunda/product-hub/issues/2860 -->
+
+### Update Elasticsearch and OpenSearch minimal supported versions to 8.16+ and 2.17+ <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
```

Comment:
@christinaausley should this not rather land in [announcements](https://docs.camunda.io/docs/next/reference/announcements-release-notes/880/880-announcements/) only? We state that announcements are used to:

> Supported environment changes and breaking changes or deprecations for the Camunda 8.8 release are summarized below.

Reply:
I agree here @MaxTru -- will move into announcements for 8.8.